docs/changelog.txt

```diff
@@ -51,7 +51,8 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - `confirm`: added a confirmation dialog for convicting dwarves of crimes
 
 ## API
-- Added ``Filesystem::mkdir_recursive``
+- Added ``Filesystem::mkdir_recursive`` (returns true if directories to create already exist)
```

Comment:
Nitpicking a bit, but I would probably avoid including implementation details in the changelog - a description of the function is fine (although I left it out earlier because I thought the name was self-explanatory), but for details about arguments, return values, etc., I would say that's what the API docs are for (in this case, `Lua API.rst` and whatever's in `Filesystem.h`).